infrastructure/variables.tf

```diff
@@ -1,15 +1,18 @@
-variable "product" {}
-
-variable "component" {}
-
-variable "location" {
-  default = "UK South"
+variable "resource_group_name" {
```

Comment:
Did you mean to remove all of the other fields that existed here?

Reply:
Thanks @jackmaloney,  yes, location, name and tags are the only arguments available when creating an Azure Resource Group, didn't want anything else confusing matters. https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_gro
up